CRAWLR/DoneDrinkingViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  DoneDrinkingViewController.swift
+//  CRAWLR
+//
+//  Created by Mayra Sanchez on 4/21/20.
+//  Copyright © 2020 Wesley Swanson. All rights reserved.
+//
+
+import UIKit
+import Contacts
+import ContactsUI
+
+import Foundation
+
+class DoneDrinkingController: UIViewController, CNContactPickerDelegate{
+
+override func viewDidLoad() {
+    super.viewDidLoad()
+    // Do any additional setup after loading the view.
+}
+    
+    @IBAction func OnPressContacts(_ sender: Any) {
+        fetchContacts()
+        let contactPicker = CNContactPickerViewController()
+        contactPicker.delegate = self
+        contactPicker.displayedPropertyKeys =
+            [CNContactGivenNameKey
+                , CNContactPhoneNumbersKey]
+        self.present(contactPicker, animated: true, completion: nil)
+        
+    }
+    
+    private func fetchContacts() {
+        print("Attempting to fetch contatcs...") //testing
+        
+        let store = CNContactStore()
+        
+        store.requestAccess(for: .contacts) { (granted, err) in
+            if let err = err {
+                print("Failed to request access", err)
+                return
+            }
+            if granted {
+                print("Access granted")
+                
+                let keys = [CNContactGivenNameKey, CNContactFamilyNameKey,
+                CNContactPhoneNumbersKey]
+                let request = CNContactFetchRequest(keysToFetch: keys as [CNKeyDescriptor])
+                
+                do {
+                    try store.enumerateContacts(with: request, usingBlock: { (contact, stopPointerifYouWantToStopEnumerating) in
+                        //print(contact.givenName, "", contact.familyName,"|", contact.phoneNumbers.first?.value.stringValue ?? "", "\n")//prints contacts
```

Comment:
You can remove this comment, also you're not doing anything in the callback function, do you even need it?